conda/environments/all_cuda-118_arch-x86_64.yaml

```
@@ -19,7 +19,7 @@ dependencies:
 - cramjam
 - cubinlinker
 - cuda-nvtx=11.8
-- cuda-python>=11.7.1,<12.0a0,<=11.8.3
+- cuda-python>=11.7.1,<12.0a0
```

What about adding an explicit exclusion for the buggy versions? I assume `11.8.3` is buggy.

```suggestion
- cuda-python>=11.7.1,<12.0a0,!=11.8.3
```

---

The buggy versions were yanked from PyPI and marked broken on conda-forge, so this is not needed. https://pypi.org/project/cuda-python/